### PR TITLE
Removed netcoreapp2.0 framework check in formatting precision tests

### PR DESCRIPTION
**Pull request by iliashkolyar**
Fixes https://github.com/fluentassertions/fluentassertions/issues/1975

* [x] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [x] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [x] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [ ] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).
* [ ] If the PR affects [the documentation](../tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).

### CODE REVIEW

**Comment by lg2de** [comment]
I added this in #1057, I guess. 
Please give me some time on Monday to verify.

**Comment by danielchalmers** [comment]
Is it worth dropping .NET Core 2.x altogether since it lost official support [over a year ago](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core)? Even 3.x is getting dropped this year.

**Comment by jnyrup** [comment]
> Is it worth dropping .NET Core 2.x altogether since it lost official support [over a year ago](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core)? Even 3.x is getting dropped this year.

v6 will keep its targets. 
For v7 we will evaluate what targets to keep/add/remove.

File: docs/_pages/releases.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ sidebar:
 
 ### Fixes
 * Fixed `For`/`Exclude` not excluding properties in objects in a collection - [#1953](https://github.com/fluentassertions/fluentassertions/pull/1953)
+* Fixed formatting precision tests that failed on netcoreapp2.0 framework for specific machines - [#1976](https://github.com/fluentassertions/fluentassertions/pull/1976)
```
Comment (IT-VBFK) [inline docs/_pages/releases.md:20]: Is this really worth to be mentioned in the release notes?

Comment (jnyrup) [reply docs/_pages/releases.md:20]: No. 
The release notes are meant for changes visible to the end user.

File: Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs [hunk 2]
```diff
@@ -458,11 +458,7 @@ public void When_formatting_single_it_should_have_max_precision()
         string result = Formatter.ToString(value);
 
         // Assert
-#if NETCOREAPP3_0_OR_GREATER
-        result.Should().Be("0.33333334F");
-#else
-        result.Should().Be("0.333333343F");
-#endif
+        result.Should().BeOneOf("0.33333334F", "0.333333343F");
```
Comment (lg2de) [inline Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs:461]: I feel uncomfortable when we don't know exactly what the expectation is.
Maybe we can argue that it is not relevant for FluentAssertion itself.

**Review by jnyrup — APPROVED** [review]

**Review by dennisdoomen — APPROVED** [review]

**Pull request merged**